ajax/libs/perfbar/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "perfbar",
+  "filename": "perfbar.min.js",
+  "version": "0.2.1",
+  "homepage": "http://lafikl.github.io/perfBar",
+  "description": "Tracking performance metrics never been easier",
+  "author": "Khalid Lafi",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WPOTools/perfBar.git"
+  },
+  "keywords": [
+    "perfbar",
+    "frontend",
+    "performance"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/WPOTools/perfBar.git",
+    "basePath": "lib",
+    "files": [
+      "*"
```

Comment:
what's the reason why to use wildcard in non-dist folder? We talk about it many times.